workloads/virtual-machines/create-networkpolicy.md

```diff
@@ -0,0 +1,76 @@
+# Enabling NetworkPolicy for VirtualMachine
+
+Before creating NetworkPolicy object, make sure you are using a networking solution which supports NetworkPolicy. Network isolation is controlled entirely by NetworkPolicy object. By default, all vms in a namespace are accessible from other vms and network endpoints. To isolate one or more vms in a project, you can create NetworkPolicy objects in that namespace to indicate the allowed incoming connections.
+
+> Note: vms and pods are equivalent regarding to NetworkPolicy
```

Comment:
> Note: vms and pods are treated equally by network policies, since labels are passed through to the pods which contain the running vm. With other words, labels on vms can be matched by `spec.podSelector` on the policy.

Not perfect either, but I think it removes some more wiggle room. It's imo essential that people get that.